src/components/badges/smallElectric.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from "react"
+
+import { BadgeProps } from "./types"
+import BaseBadge from "./base"
+import ElectricIcon from "../../assets/dist/electric"
+
+export const SmallElectricBadge: FC<BadgeProps> = ({
```

Comment:
They will look very similar, the only difference is the size.
Small : 
<img width="95" alt="samll" src="https://user-images.githubusercontent.com/52455155/140029521-2dd24584-b026-4fb9-8aaf-99d90dc5ddf1.png">
Large : 
<img width="100" alt="large" src="https://user-images.githubusercontent.com/52455155/140029537-98f4202c-5824-42ee-883b-9e3a29492e32.png">

Reply:
Since you plan a breaking change for the withText. I suggest switching the approach and having a `LargeElectricBadge` version for the current one as the small version looks like all the other bages. So we would be aligned and the one that is different is the separate component

Reply:
Yes, make sense !